src/category_theory/elements.lean

```diff
@@ -1,17 +1,31 @@
-import category_theory.types
-import category_theory.comma
 import category_theory.equivalence
+import category_theory.comma
 import category_theory.punit
 import category_theory.eq_to_hom
 
+/-!
```

Comment:
The new style guides call for splitting the copyright header (before imports) and the module doc (after imports). Again, this is kind of arbitrary, but Lean doesn't accept module docs before the import, and to preserve some similarity to core, I thought it was nice to keep the copyrights at the very top.

Could you expand the module doc string to fit the new (soon to be merged) guidelines?

Reply:
I've expanded this. Please have a look and let me know what you think.

Reply:
I think it looks great, thanks!